style.css

```diff
@@ -1,37 +1,263 @@
 .head {
-  font-size: 20px;
+    font-weight: bold;
+    font-size: 30px;
+}
+
+h1,
+h2,
+h3 {
+    padding-bottom: 12px;
+}
+
+.job,
+.education {
+    margin-bottom: 5px;
+}
+
+.job h3 {
+    margin-bottom: -16px;
+    margin-top: 30px;
+
+}
+
+.job p {
+    line-height: 1.2;
+
+}
+
+hr {
+    margin: 20px 0;
 }
 
 a {
-  color: black;
-  text-decoration: none;
+    color: #333;
+    text-decoration: none;
 }
 
-@media only screen and (max-width: 430px) {
-  .left,
-  .right {
-    display: none;
-  }
-  .stuff {
-    width: 100%;
+a:hover {
+    color: #666;
+    text-decoration: underline;
+}
+
+ul {
+    padding: 0;
+    list-style-type: none;
+}
+
+.social-icons {
+    display: flex;
+    justify-content: flex-start;
+    margin-top: -36px;
+    margin-left: 127px;
+}
+
+.github-icon,
+.linkedin-icon {
+    position: relative;
+    text-align: left;
+}
+
+.github-icon {
+    left: -10px;
+    top: 0px;
+}
+
+.linkedin-icon {
+    left: -10px;
+    top: 0px;
     margin-left: 10px;
-  }
+}
+
+.icon {
+    width: 32px;
+    height: 32px;
+    transition: all 0.3s ease;
+}
+
+.github-icon:hover .icon,
+.linkedin-icon:hover .icon {
+    transform: scale(1.2);
+    opacity: 0.7;
+}
+
+.blog-link {
+    text-decoration: none;
+    color: #333;
+    letter-spacing: normal;
+    transition: all 0.3s ease;
+}
+
+.blog-link:hover {
+    letter-spacing: 1px;
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+}
+
+.education-link {
+    text-decoration: none;
+    color: #333;
+    letter-spacing: normal;
+    transition: letter-spacing 0.3s ease;
+}
+
+.education-link:hover {
+    letter-spacing: 1px;
+}
+
+.focus-areas {
+    padding: 6px;
+    border: 1px solid #f5f5f5;
+    background-color: #f5f5f5;
+    font-size: 14px;
+
+}
+.title-focus {
+    color: #333;
+    font-size: 14px;
+    font-weight: bold;
+    margin-bottom: 10px;
+}
+
+.blog-title {
+    font-size: 30px;
+    color: #333;
+}
+
+.tooltip {
+    display: none;
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0.8);
+    color: white;
+    padding: 5px;
+    border-radius: 3px;
+    font-size: 14px;
+    z-index: 100;
+}
+
+li {
+    position: relative;
+    cursor: pointer;
+}
+
+
+
+
+.focus-areas span {
+    display: inline-block;
+    transition: transform 0.3s ease;
+}
+
+
+
+/* Стиль кнопок тегов */
+.filter-tags {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 6px;
+    margin-top: -36px;
+    margin-bottom: 10px;
+}
+
+.tag-button {
+    background-color: #f5f5f5;
+    border: 1px solid #ccc;
+    color: #333;
+    padding: 5px 10px;
+    border-radius: 3px;
+    font-size: 14px;
+    cursor: pointer;
+    text-decoration: none;
+    transition: all 0.3s ease;
+}
+
+.tag-button:hover {
+    background-color: #333;
+    color: #f5f5f5;
+    border: 1px solid #333;
+}
+
+.tag-button:focus {
+    outline: none;
+}
+
+.tag-button:active {
+    background-color: #555;
+    border: 1px solid #555;
+}
+
+/* Стиль для экранов шириной до 600px */
```

Comment:
ditto

Reply:
Here, I left a note for myself before the media query when I was creating the responsive design. I've removed it from the final code, but I've described the logic in the readme